### PR TITLE
APPS-5652: SPA dialogs

### PR DESCRIPTION
**Pull request by mharacewiat**
### !!! could be merged when https://spryker.atlassian.net/browse/APPS-5652 is done !!!

Adjusting documentation to the new manifest schema.

## Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/acp/user/app-manifest.md [hunk 1]
```diff
@@ -210,6 +220,23 @@ For the manifest, make sure to follow these conditions:
...
+        "type": "text",
+        "data": "bar"
+    },
+    "baz": {
+        "type": "text",
+        "data": "qux"
+    }
+}
+```
+</details>
```
Comment (matweew) [inline docs/acp/user/app-manifest.md:238]: Imagine you are a PBC developer and need to provide a manifest for your new app. 
You read this document and see `messages` with some key, type and data. What can you put into the values? is it required? does my application need it?

Can you try to explain on the examples how these `messages` could be used.

**Review by matweew — CHANGES_REQUESTED** [review]

File: docs/acp/user/app-manifest.md (hunk 1 above)
Comment (mharacewiat) [reply docs/acp/user/app-manifest.md:238]: That's an optional property. In such case you skip it.

Comment (matweew) [reply docs/acp/user/app-manifest.md:238]: The question is HOW to use it. Where is the list of available `key`s (onConnect, connectSuccessful or other).
What `type` are available?

Comment (mharacewiat) [reply docs/acp/user/app-manifest.md:238]: - Keys should be documented on PBC end, they define what they need.
- The only type supported right now is `text`. That was mentioned in the table.
    > Mandatory. Message type (available types: `text`)

File: docs/acp/user/app-manifest.md [hunk 2]
```diff
@@ -69,33 +69,43 @@ For the manifest, make sure to follow these conditions:
...
+| messages         | Optional. An array of objects represented as application messages. The message object has following keys: <ul><li>`type` - Mandatory. Message type (available types: `text`).</li><li>`data` - Mandatory. Message data.</li></ul>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      | See *message example* under this table.
```
Comment (matweew) [inline docs/acp/user/app-manifest.md:108]: What is message data? is it text or JSON? if it's an object what properties are needed? Would be nice to have an example in the example section.

File: docs/acp/user/app-manifest.md (hunk 1 above)
Comment (Serhii-SPRYKER) [reply docs/acp/user/app-manifest.md:238]: @mharacewiat 
Right now only two keys from the FE side are supported

onSave, onDisconnect

They could not be defined by PBC side

Comment (mharacewiat) [reply docs/acp/user/app-manifest.md:238]: @matweew What do we do with it?

Comment (matweew) [reply docs/acp/user/app-manifest.md:238]: add them to the description of messages and examples

File: docs/acp/user/app-manifest.md (hunk 2 above)
Comment (matweew) [reply docs/acp/user/app-manifest.md:108]: @mharacewiat @Serhii-SPRYKER there is also the picture at the end of the page which shows where the parts of manifests are used. So please add a picture with the example where the new feature is used (config page -> dialogs)

**Comment by pushokwhite** [comment]
https://github.com/spryker/spryker-docs/pull/1569#pullrequestreview-1274733763

File: docs/acp/user/app-manifest.md (hunk 1 above)
Comment (pushokwhite) [reply docs/acp/user/app-manifest.md]: They have been added, but I lack clarification - when they trigger and what to expect if we don't define them in the manifest.

**Review by matweew — COMMENTED** [review]
@mharacewiat @Serhii-SPRYKER there is also the picture at the end of the page which shows where the parts of manifests are used. So please add a picture with the example where the new feature is used (config page -> dialogs)

**Review by matweew — DISMISSED** [review]

**Review by pushokwhite — DISMISSED** [review]

**Pull request merged**